frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -205,7 +205,11 @@ class Slider extends React.PureComponent<Props, State> {
     if (slider && thumb) {
       const sliderPosition = slider.getBoundingClientRect()
       const thumbPosition = thumb.getBoundingClientRect()
+      const thumbContainer = thumb.parentElement
 
+      if (thumbContainer)
+        thumbContainer.style.left =
```

Comment:
This change seems to have an impact on the position that gets selected in our e2e slider tests. For example, this test currently results in the `26` selected, but with this change `25` gets selected instead: 

https://github.com/streamlit/streamlit/blob/2d09706c27b4f2daa1c05385ac45a30e594e44a6/e2e/specs/st_slider.spec.js#L83

I'm not sure why this is happening. However, we can just update the values in the e2e test to reflect this change. But we should make sure that this is really the expected behavior.

Reply:
btw. all the e2e tests that are failing are related to this slider issue

Reply:
Thanks for taking a look and confirming, @LukasMasuch! Let me know if updating the e2e tests is feasible, or if we don't want to introduce this change because it breaks its behavior. It is sort of a _hacky_ way to fix the thumb positioning tbh, because there's no native way to fix it using baseweb, so happy to revert it if it's not something we'd like to add to the codebase.